main.go

```diff
@@ -239,18 +252,31 @@ func run(c *cli.Context) error {
 	// Setup execution environment
 	environ := os.Environ()
 	environ = append(environ, fmt.Sprintf("GOOGLE_APPLICATION_CREDENTIALS=%s", keyPath))
+	environ = append(environ, fmt.Sprintf("KUBECONFIG=%s", kubeconfigPath))
 	runner := NewBasicRunner("", environ, os.Stdout, os.Stderr)
 
-	// Auth with gcloud and fetch kubectl credentials
-	if err := fetchCredentials(c, project, runner); err != nil {
-		return err
+	if kubeconfigCredentials {
+		// Load kubeconfig from environment
+		if err := loadKubeconfig(c, project, runner); err != nil {
+			return err
+		}
+	} else {
+		// Auth with gcloud and fetch kubectl credentials
+		if err := fetchCredentials(c, project, runner); err != nil {
+			return err
+		}
 	}
 
 	// Delete credentials from filesystem when finishing
 	// Warn if the keyfile can't be deleted, but don't abort.
 	// We're almost certainly running inside an ephemeral container, so the file will be discarded when we're finished anyway.
 	defer func() {
-		err := os.Remove(keyPath)
+		if kubeconfigCredentials {
+			err = os.Remove(kubeconfigPath)
+		} else {
+			err = os.Remove(keyPath)
```

Comment:
dit moet `:=` worden want de `err` variable is hier nog niet defined

Reply:
Ja, ik probeerde dat, kreeg dit:

![image](https://user-images.githubusercontent.com/226540/96986889-8e92f880-1522-11eb-9d2f-4e2eb3629ea3.png)

```
 github.com/NYTimes/drone-gke
./main.go:275:4: err declared and not used
./main.go:277:4: err declared and not used
Makefile:139: recipe for target 'drone-gke' failed
```

Reply:
So then I tried this: 

![image](https://user-images.githubusercontent.com/226540/96986974-a8344000-1522-11eb-9171-bcec85cce2f4.png)

En kreeg ik:
```
# github.com/NYTimes/drone-gke
./main.go:274:7: use of untyped nil
Makefile:139: recipe for target 'drone-gke' failed
make: *** [drone-gke] Error 2
```

Reply:
_Go revolution cancelled_

Reply:
heh

Reply:
en what about ipv `err := nil` -> `var err error`

Reply:
GENIUS